i18n-config.js

```diff
@@ -73,11 +73,11 @@ module.exports = {
         short_name: '繁體',
         affiliate_lang: 11,
     },
-    ach: {
-        path: 'ach',
-        display_name: 'Crowdin',
-        short_name: 'CR',
-    },
+    // ach: {
+    //     path: 'ach',
+    //     display_name: 'Crowdin',
+    //     short_name: 'CR',
+    // },
```

Comment:
@mohammad-rework any particular reason for this?

Reply:
![Screenshot 2022-02-09 at 12 15 06 PM](https://user-images.githubusercontent.com/93753441/153382290-839b182e-bb5a-464c-a4ad-950e6e61f703.png)
It was showing in the test link!

Reply:
This is intentional and is removed for production. Please revert that change in this PR. @mohammad-rework